topological_navigation/src/topological_navigation/policies.py

```diff
@@ -76,10 +81,12 @@ def _update_everything(self) :
         print "All Done"
 
 
-    def create_edge(self, point1, point2):
+    def create_edge(self, point1, point2, color):
```

Comment:
in general, would be better to define defaults for new parameters to keep the API backwards-compatible if possible, e.g. `color='red'`.